src/announcements/tests/test_views.py

```diff
@@ -74,12 +72,12 @@ def test_all_announcement_page_status_codes_for_students(self):
             expected_url=reverse('announcements:list', args=[self.ann_pk]),
         )
 
-        # These views should redirect to admin login
-        self.assertRedirectsAdmin('announcements:create')
-        self.assertRedirectsAdmin('announcements:delete', args=[1])
-        self.assertRedirectsAdmin('announcements:update', args=[1])
-        self.assertRedirectsAdmin('announcements:copy', args=[1])
-        self.assertRedirectsAdmin('announcements:publish', args=[1])
+        # These views should redirect to login
```

Comment:
Comment is a lie? Says they should redirect to login, but we are testing for 403?

Should probably say "Authenticated users that aren't staff should get permission denied 403

Reply:
fixed